library/Garden/Exception/ValidationException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @author Todd Burry <todd@vanillaforums.com>
+ * @copyright 2009-2014 Vanilla Forums Inc.
+ * @license MIT
+ */
+
+namespace Garden\Exception;
+
+use Garden\Validation;
+
+/**
+ * An exception that was built from a {@link Validation} object.
+ *
+ * The validation object collects errors and is mutable. Once it's ready to be thrown as an exception it gets converted
+ * to an instance of the immutable {@link ValidationException} class.
+ */
+class ValidationException extends ClientException {
```

Comment:
Oh shoot. Did not realize this would bring in the ClientException which I don't think I'm ready for. Can you refactor this to just extend \Exception for now and then remove ClientException and Response?